manipmongo/manipulator.go

```diff
@@ -62,11 +62,14 @@ func New(url string, db string, options ...Option) (manipulate.TransactionalMani
 
 	if cfg.tlsConfig != nil {
 		dialInfo.DialServer = func(addr *mgo.ServerAddr) (net.Conn, error) {
-			return tls.Dial("tcp", addr.String(), cfg.tlsConfig)
+			d := net.Dialer{Timeout: dialInfo.Timeout}
+			return tls.DialWithDialer(&d, "tcp", addr.String(), cfg.tlsConfig)
 		}
 	} else {
 		dialInfo.DialServer = func(addr *mgo.ServerAddr) (net.Conn, error) {
-			return net.Dial("tcp", addr.String())
+			//return net.Dial("tcp", addr.String())
+			// TODO: add connect timeout 
```

Comment:
Do you need to keep the comments? I feel like we are adding timeout and the function is simple enough to understand. I think we can remove these two lines.

Reply:
no, I don't need to keep comments